cmd/start.go

```diff
@@ -54,6 +54,7 @@ func init() {
 	startCmd.Flags().String("crd-group", "", "the group of the CRD you want monitored (ex: stable.wpengine.io)")
 	startCmd.Flags().String("crd-version", "v1", "the version of the CRD you want monitored")
 	startCmd.Flags().String("crd-namespace", metav1.NamespaceNone, "(optional) the namespace of the CRD you want monitored, only needed for namespaced CRDs (ex: default)")
+	startCmd.Flags().String("crd-optIn", "", "(optional) Annotation key to specify that the custom resource has opted in to watching by Lostromos")
```

Comment:
To document our conversation around this, `crd-filter` appears to be a better name for this.